scripts/journalscraper.sh

```diff
@@ -8,7 +8,7 @@ while true;
 do
 	now=$(date +"%F %T")
 
-	journalctl -u bisq-pricenode --since="$last" --until="$now" | grep -Eo "getAllMarketPrices.*bisq/[0-9].[0-9].[0-9]" | cut -d / -f 2 | sort | uniq -c | while read -r line; do
+	journalctl -u bisq-pricenode --since="$last" --until="$now" | grep -Eo "getAllMarketPrices.*bisq/[0-9]*\.[0-9]*\.[0-9]*" | cut -d / -f 2 | sort | uniq -c | while read -r line; do
```

Comment:
I would do "+" instead of "*". That's a more exact match.
"getAllMarketPrices.*bisq/[0-9]+\.[0-9]+\.[0-9]+"

Reply:
Change applied, thanks!  Is it ok to match the period explicitly rather than any char?

Reply:
Should be fine. As long as we don't change the version shema.